zen_queries/rest_framework.py

```diff
@@ -24,4 +32,7 @@ def get_serializer(self, *args, **kwargs):
         )
         if self.request.method == "GET":
             serializer = disable_serializer_queries(serializer)
+            if getattr(serializer, "many", False):
+                # Serializer data must be fully evaluated prior to serialization. See #18.
```

Comment:
Not sure about putting links to issues in the code..

Reply:
I could just remove the reference 
```suggestion
                # Serializer data must be fully evaluated prior to serialization.
```

But really not sure what that gains. It can be very useful to track down which issue code is for. At first look this  block does look a little suspect/

Reply:
OK - happy to leave it in if you think it helps.